modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/dev/public/js/static-env-utils.js

```diff
@@ -59,5 +59,19 @@ const Liferay = {
 
 window.defaultFetch = fetch;
 
+window.fetch = (resource, {headers, ...init} = {}) => {
+	headers = new Headers({
+		Accept: 'application/json',
+		Authorization: `Basic ${window.btoa('test@test.com:test')}`,
+		'Content-Type': 'application/json',
+	});
+
+	// eslint-disable-next-line @liferay/portal/no-global-fetch
+	return window.defaultFetch(resource, {
```

Comment:
Hey Giu, Why are you overwriting browser's default fetch?

Reply:
because in my static env i't not possible to test api fetch, i figure out as not authenticated